osvr_tracked_device.h

```diff
@@ -575,7 +509,7 @@ float OSVRTrackedDevice::GetFloatTrackedDeviceProperty(vr::TrackedDeviceProperty
     case vr::Prop_UserIpdMeters_Float: // TODO
         if (error)
             *error = vr::TrackedProp_Success;
-        return static_cast<float>(m_DisplayConfiguration->getIPDMeters());
+        return 0.063f; /// @todo fix this once OSVR API has IPD
```

Comment:
I modified this to just call GetIPD(). Working on the distance between eye poses now.